py/torch_tensorrt/fx/passes/lower_basic_pass_aten.py

```diff
@@ -437,8 +438,10 @@ def compose_bmm(
             real_other = input_other_n.all_input_nodes[0]
             if len(real_other.meta["val"].size()) == 2:
                 new_func = aten_compose_bmm_2d
-            if len(real_other.meta["val"].size()) == 3:
+            elif len(real_other.meta["val"].size()) == 3:
                 new_func = aten_compose_bmm_3d
+            else:
```

Comment:
Not clear why we need this `new_func = torch.ops.aten.matmul`? Any example or unit test?

Reply:
This addition is related to an issue in the `compose_bmm` lowering pass. I noticed that [_input_n_](https://github.com/pytorch/TensorRT/blob/cefb2f2280e0b9f4dbc766103c0c2be5d9803a1c/py/torch_tensorrt/fx/passes/lower_basic_pass_aten.py#L417) can have a different shape than [_real_input_](https://github.com/pytorch/TensorRT/blob/cefb2f2280e0b9f4dbc766103c0c2be5d9803a1c/py/torch_tensorrt/fx/passes/lower_basic_pass_aten.py#L428), which causes the batch matrix multiply to have 4 dimensions instead of 3, reaching this `else` statement.